src/esp/metadata/attributes/PhysicsManagerAttributes.cpp

```diff
@@ -14,6 +14,7 @@ PhysicsManagerAttributes::PhysicsManagerAttributes(const std::string& handle)
   setTimestep(0.008);
   setGravity({0, -9.8, 0});
   setFrictionCoefficient(0.4);
+  setRestitutionCoefficient(0.1);
```

Comment:
Separate PR?

Reply:
Need to have initial values for all supported values for this PR, since there's no default value available/provided.